src/libstd/sys/windows/handle.rs

```diff
@@ -46,7 +46,8 @@ impl Handle {
     pub fn into_raw(self) -> c::HANDLE {
         let ret = self.raw();
         mem::forget(self);
-        return ret;
+
+        ret
```

Comment:
nit: the extra newlines in here are surprising to me.  (They're more reasonable after close braces, but in a three-line function the blank line doesn't seem to be adding anything.)

Reply:
I tried to keep consistent but yeah it looks weird.
Should I revert it everywhere?